bitnami/postgresql-ha/templates/backup/cronjob.yaml

```diff
@@ -66,7 +66,7 @@ spec:
                     {{- if (include "postgresql-ha.postgresqlCreateSecret" .) }}
                     key: password
                     {{- else }}
-                    {{- if ( index ( lookup "v1" "Secret" (include "common.names.namespace" .) (include "postgresql-ha.postgresqlSecretName" .) ) ".data.postgres-password" ) }}
+                    {{- if ( index ( lookup "v1" "Secret" (include "common.names.namespace" .) (include "postgresql-ha.postgresqlSecretName" .) ) "data.postgres-password" ) }}
```

Comment:
Why do you need this change instead of keeping it as in `witness-statefulset.yaml` or `statefulset.yaml`?